src/cmp_aus.js

```diff
@@ -95,31 +95,80 @@ const reloadPage = async (page) => {
 };
 
 const checkPrebid = async (page) => {
-	log(`Reloading page: Start`);
+	// --------------- RELOAD PAGE START ---------------------------
+	log(`[TEST 4: RELOAD PAGE] Step start`);
 	const reloadResponse = await page.reload({
 		waitUntil: 'domcontentloaded',
 		timeout: 30000,
 	});
 	if (!reloadResponse) {
-		logError(`Reloading page : Failed`);
+		logError(`[TEST 4: RELOAD PAGE] Reloading page : Failed`);
 		throw 'Failed to refresh page!';
 	}
-	log(`Reloading page: Complete`);
+	log(`[TEST 4: RELOAD PAGE] Step complete`);
+	// --------------- RELOAD PAGE END ---------------------------
+
+	// --------------- BUNDLE START ---------------------------
+	log(`[TEST 4: PREBID BUNDLE] Checking: graun.Prebid.js.commercial.js`);
+	await page.waitForRequest((req) =>
+		req.url().includes('graun.Prebid.js.commercial.js'),
+	);
+	log(`[TEST 4: PREBID BUNDLE] Step start`);
+	// --------------- BUNDLE END ---------------------------
 
+	// --------------- PAGESKIN START ---------------------------
+	log(`[TEST 4: PAGESKIN] Step start`);
 	const hasPageskin = await page.evaluate(() =>
 		document.body.classList.contains('has-page-skin'),
 	);
 
 	if (hasPageskin) {
-		log('Pageskin detected. Prebid will not run');
+		log('[TEST 4: PAGESKIN] Pageskin detected. Prebid will not run');
 		return Promise.resolve();
 	}
+	log(`[TEST 4: PAGESKIN] Step complete`);
+	// --------------- PAGESKIN END ---------------------------
 
+	// --------------- PUBMATIC START ---------------------------
+	log(`[TEST 4: PUBMATIC] Step start`);
 	const prebidURL =
 		'https://hbopenbid.pubmatic.com/translator?source=prebid-client';
 
 	await page.waitForRequest((req) => req.url().includes(prebidURL));
-	log(`Prebid check: Complete`);
+	log(`[TEST 4: PUBMATIC] Step complete`);
+	// --------------- PUBMATIC END ---------------------------
+
+	// --------------- PBJS START ---------------------------
+	log(`[TEST 4: PBJS] Step start`);
+	const hasPrebid = await page.evaluate(() => window.pbjs !== undefined);
```

Comment:
I think you'd benefit from using [`page.waitForFunction`](https://pptr.dev/api/puppeteer.page.waitforfunction) instead of `page.evaluate` like we've done in the E2E tests e.g. https://github.com/guardian/commercial/blob/main/playwright/tests/prebid.spec.ts#L83

Both puppeteer and playwright seem to support this